src/index.js

```diff
@@ -1,28 +1,98 @@
-import _ from "lodash";
-import printMe from "./print.js";
-import "./style.css";
-import Icon from "./icon.png";
+import './style.css';
 
-function component() {
-  const element = document.createElement("div");
-  const btn = document.createElement("button");
+const tasks = [];
 
-  //Lodash, now imported by this script
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
-  element.classList.add("hello");
+const todoList = document.getElementById('todo-list');
+const addTaskButton = document.getElementById('add-task-button');
+const newTaskInput = document.getElementById('new-task-input');
+const clearAllButton = document.getElementById('clear-all-button');
 
-  btn.innerHTML = "Click me and check the console!";
-  btn.onclick = printMe;
+const task1 = {
+  description: 'Complete JavaScript project',
+  completed: false,
+};
```

Comment:
- [ ] As shown in the image below detailing the projects requirement, each task is expected to have three keys (description, completed, index), you currently have two (description, completed). please i kindly suggest you add the third key which is `index` for every task. see the example below for how it should be done
- > project requirement
![miss](https://github.com/Peter-Kibet/ToDo_List/assets/89055502/09e9f25c-1c0f-442d-9034-256f70029ee7)
- > example code 
```javascript
{
    description: 'An example description in string',
    completed: false,
    index: 1,
  },

```